language/README.md

```diff
@@ -31,30 +31,30 @@ Learning API.
 
 View the [documentation][analyze_docs] or the [source code][analyze_code].
 
-__Usage:__ `node analyze --help`
+__Usage:__ `node analyze.js --help`
 
 ```
 Commands:
-  sentimentOfText <text>               Detect the sentiment of a block of text.
```

Comment:
It's not the fault of the program, it's the fault of google-cloud-node. There's nothing I can fix about that in this PR.